ffigen/README.md

```diff
@@ -0,0 +1,14 @@
+This package configures the ffigen to generate bindings to realm-core library.
+The is executed manually as needed and this package is never published.
+
+Usage: 
+
+On Windows: dart run ffigen --config windows.yaml
+On MacOS: dart run ffigen --config macos.yaml
+On Linux: dart run ffigen --config linux.yaml
+
+Note: Dart ffigen tool generated platform dependent bindings which are tailored for the host platform the ffigen is executed. Currently all of platforms generate similar output with the configuration workaround here: https://github.com/dart-lang/ffigen/pull/119. 
```

Comment:
it is by chance right now that mac and linux can use same config. Linux is can't. These are configurations for the host platform and not the target platform so I suggest we leave them in. If we don't need them in the future we could remove them. These are temporary anyways since we are waiting for the platform independent ffi generator